src/main/java/org/commcare/xml/IndexedFixtureXmlParser.java

```diff
@@ -90,7 +90,10 @@ private void processRoot(TreeElement root) throws IOException {
                 processEntry(entry, indices);
             }
         } else {
-            sandbox.getIndexedFixtureStorage(fixtureName).removeAll();
+            IStorageUtilityIndexed storage = sandbox.getIndexedFixtureStorage(fixtureName);
```

Comment:
Confirming that even if the user is not assigned any location, a fixture for that location is present with no children/rows in restore ?

Reply:
That's right, it should look like this:
```xml
<fixture id="locations" indexed="true" user_id="c0bdfa6db8ef44b8af0c9fbb70e4e19e">
  <locations/>
</fixture>
```
Though the fixture isn't present for all domains - only those that actually use locations